src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -675,10 +680,12 @@ UCS_CLASS_INIT_FUNC(uct_rc_mlx5_iface_common_t,
         goto cleanup_dm;
     }
 
-    status = uct_rc_mlx5_devx_init_events(self);
+#if HAVE_DEVX
```

Comment:
why do we need `#if HAVE_DEVX`?
wouldn't it just reutrn UCS_OK if no devx?

Reply:
symbol uct_rc_mlx5_devx_iface_init_events only available when we have devx